invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/details/browse/index.html

```diff
@@ -12,6 +12,7 @@
 {% set active_community_header_menu_item= 'browse' %}
 {% set subcommunities_endpoint = community["links"]["subcommunities"] %}
 {% set community_slug = community["slug"] %}
+{% set children_allowed = community["children"]["allow"] %}
```

Comment:
There is already a [_show_browse_page here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/communities_ui/views/ui.py#L71): why is it not working correctly?
Is it in the case you have a direct link?

Reply:
The `_show_browse_page` is for whether the `Browse` link should appear in the menu. This variable `children_allowed` is toggling whether we should link to the subcommunities react search kit page. When children aren't allowed this page 404s